README.rst

```diff
@@ -16,7 +16,7 @@ Setup
 
 .. code:: bash
 
-    pip install ramos
+    make install
```

Comment:
I think the *setup* should remain the way it was, since `pip install ramos` is the way you can download it to your app.

You could create a new section ("Development setup"?) with `make  install`.

Reply:
@renanivo Tks for the tip... I adjusted the `README`.